src/Compilers/CSharp/Test/Semantic/Semantics/RefExtensionMethodsTests.cs

```diff
@@ -38,6 +38,35 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_RefLvalueExpected, "5").WithLocation(13, 9));
         }
 
+        [Fact]
+        public void ExtensionMethods_StructCollectionInitializer()
```

Comment:
Should add a test where `this` is passed by `ref` and one of the arguments is passed by `in`. Use a custom struct for the latter parameter. Want to make sure the implicit `in` is handled correctly in that case.

Reply:
How should I verify the correct usage of implicit `in`? Is it necessary to verify IL?

Reply:
Added a test which passes `in MyStruct` as the "element" parameter of `Add`.
- One with an explicit temp
- One where the compiler creates a temp
Verified that the struct is passed by ref in both cases.